game/screen_paths.cc

```diff
@@ -98,7 +98,7 @@ void ScreenPaths::generateMenuFromPath(fs::path path) {
 	
 	// Extract list of all directories
 	std::list<fs::path> directories;
-	for (const auto &di : fs::directory_iterator(path)) {
+	for (const auto &di : fs::directory_iterator(path, fs::directory_options::skip_permission_denied)) {
```

Comment:
seems that this option interrupts the iterating; passing an error code seems to allow to continue iterating